### DISCUSSION TITLE
Compare fleetbase  better

### DISCUSSION THREAD

**Discussion by Jengro777** [karrioapi/karrio#403] [Q&A]
https://github.com/fleetbase/fleetbase

**Comment by danh91** [comment]
Hi @Avey777,
If I understand correctly, you are asking for a comparison of [Karrio](https://github.com/karrioapi/karrio) with [Fleetbase](https://github.com/fleetbase/fleetbase)?

Karrio and Fleetbase are both open-source solutions for the supply chain industry but we are solving different problems for different types of users.

## Fleetbase (to my knowledge) is primarily a **Delivery management system** 

### With Fleetbase you can:

- Manage aspects of delivery operations such as Booking, Pickup, Dropoff
- Manage delivery drivers and routing optimization
- Manage delivery booking with rating (delivery costs)
- Offer real-time tracking for delivery operations
- Have customers book delivery services
- ....

### Fleetbase typical users are:

- Last-mile carriers with a fleet of drivers
- 3PL that have their own delivery fleet
- Retailers that have their own delivery fleet
- Cargo carriers
- ....

_I recommend you talk to @shivthakker and the Fleetbase team to get their own perspective_

## Karrio on the other hand is a **Shipping management system**

### With Karrio you can:

- Manage a network of shipping carriers for your fulfilment operations (Karrio has integrations with multiple carriers like UPS, FedEx, USPS....)
- Compare shipping rates for various carriers
- Buy and generate shipping labels from carriers
- Access package tracking information
- Make real-time shipping rates accessible to your customers
- ....

### Karrio typical users are:

- eCommerce and retailers who offer online shopping with shipping options 
- Manufacturer who offers DTC shipments
- Logistics providers and 3PL that need large-scale fulfillment automation
- ....

My current understanding is that **Fleetbase** focuses on **couriers and delivery providers** whereas **Karrio** focuses on **shippers and Logistics providers**.


Note that we have some overlapping features but serve different users. Also, we are both developer-first whereby unlike the existing solutions in the industry, we are empowering developers to build supply chain solutions tailored to their business needs.
You can self-host, fork and build on top of Karrio and Fleetbase for your custom logistics operation needs.
